addon/components/md-nav.js

```diff
@@ -0,0 +1,19 @@
+import Ember from 'ember';
+import layout from '../templates/components/md-nav';
+
+const { run: { next }, Component } = Ember;
+
+export default Component.extend({
+  classNames: ['md-nav'],
+  layout,
+  tagName: 'nav',
```

Comment:
>  I was encountering a bug with liquid-fire where position:fixed does not mesh well with .liquid-container transformY properties. 

This may be a limitation rather than a bug. `liquid-container` has a `transform(0)` to support older browsers (IE9?). Can't recall the particulars, but I'm sure they can be google'd

> I have worked around the issue by placing my nav outside the outermost liquid-outlet.

wormholing is an option, but make sure the stacking context works out in the way you intend.

> On the original note, I wonder if the component should be a wrapper that can accept `fixed=true`, and the nav be a child element?

non-fixed nav use cases don't use the wrapping div. The current implementation feels right to me.